tirepressure.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TIREPRESSURE_H
+#define TIREPRESSURE_H
+
+#include "ui_mainwindow.h"
+
+class TirePressure
+{
+public:
+    TirePressure();
+    void setup(Ui::MainWindow*);
+    void front_left_bad();
```

Comment:
Can you enum the values so that you only have 2 functions instead of r for each tire. Check out gears class